tests/devices/test_default_gaussian.py

```diff
@@ -788,57 +788,3 @@ def circuit():
         ):
             circuit()
         assert dev.shots == sum(shots)
-
```

Comment:
Deleted this one, because we have a `test_apply` that should test the same thing on a lower level.